abi/include/_bits/errno.h

```diff
@@ -41,9 +41,9 @@
 #define _BITS_ERRNO_H_
 
 #ifdef __OPAQUE_ERRNO__
-#include <_bits/opaque_handle.h>
+#include <_bits/__opaque_handle.h>
```

Comment:
It seems to me you changed the include directive but didn't actually rename the file _bits/opaque_handle.h to _bits/__opaque_handle.h, or am I missing something?

Reply:
You're right, good catch!